catroid/src/main/java/org/catrobat/catroid/ui/ProjectActivity.java

```diff
@@ -251,7 +251,10 @@ public void onPositiveButtonClick(DialogInterface dialog, String textInput) {
 							File imageDirectory = new File(currentScene.getDirectory(), IMAGE_DIRECTORY_NAME);
 							File file = StorageOperations
 									.copyUriToDir(getContentResolver(), uri, imageDirectory, lookFileName);
-							sprite.getLookList().add(new LookData(textInput, file));
+
+							LookData lookData = new LookData(textInput, file);
+							sprite.getLookList().add(lookData);
+							lookData.getCollisionInformation().calculate();
```

Comment:
how long is this going to take in worst case scenario since this is run from ui thread here and in spriteActivity?

Reply:
since the loaded file has a reall small height and width the computation is instantly finished